README.md

```diff
@@ -1,2 +1,87 @@
-# antrea-operator
-Antrea Operator for Openshift
+# Antrea Operator
+
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
+## Overview
+
+The Antrea Operator installs [Antrea](https://github.com/vmware-tanzu/antrea/)
+CNI plugin on an OpenShift Kubernetes cluster.
+The operator will be deployed in the early phases of cluster
+deployment, and it will take care of deploying Antrea components:
+
+* The antrea-controller deployment
+* The antrea-agent daemonset
+
+For Openshift 4 clusters, the antrea-operator monitors the network CR in the
+config.openshift.io namespace to update the container network CIDRs used by
```

Comment:
Do we need to mention that operator also monitors operator crd for the multus config

Reply:
Currently we don't support change `DisableMultiNetwork` option.

I checked the codes of project `openshift/cluster-network-operator`, the change `DisableMultiNetwork` option is not handled. I'm not sure if we need to support such case. I create an issue https://github.com/ruicao93/antrea-operator/issues/14 to track the issue.